apps/scripts/src/limit-redis.js

```diff
@@ -15,11 +15,62 @@ const { Logger } = require("@statsify/logger");
 const { MetadataScanner, Player, Guild } = require("@statsify/schemas");
 
 const CONSTRUCTORS = [Player, Guild];
+const HISTORICAL_CONSTRUCTORS = [
+  ["DAILY", Player],
+  ["WEEKLY", Player],
+  ["MONTHLY", Player],
+];
 
 const logger = new Logger("Redis Limiter");
 const redis = new Redis(process.env.REDIS_URL);
 
-const limit = async () => {
+const limitHistorical = async () => {
```

Comment:
`limitHistorical` and `limitNormal` share 90% of the implementation. Make another function that just takes in the prefix for the leaderboard keys to limit a leaderboard and call it in these functions instead of writing the implementation twice.